docs/desktop-feature-api.md

```diff
@@ -77,24 +90,58 @@ XPCOMUtils.defineLazyModuleGetters(this, {
 
 `isEnabled({sendExposureEvent = false}): boolean`
 
-Returns a value representing the "enabled" state of the feature. You can optionally send an exposure event when the function is called.
+Short form for `.getVariable("enabled")`
+
+Note that you should have an `enabled` property defined in the manifest if you plan to use this.
 
 ```js
 const isEnabled = NimbusFeatures.myFeature.isEnabled();
 ```
 
-### `getValue()`
+### `getVariable()`
 
-`getValue({sendExposureEvent = false}): FeatureValue`
+`getVariable(variableName: string, {sendExposureEvent = false}): FeatureValue`
 
-Returns the value of all feature variables. You can optionally send an exposure event when the function is called.
+Returns the value of a single feature variable. You can optionally send an exposure event when the function is called.
+
+Warning: **This function will throw in Nightly and CI build** if you do not define `variableName` in the Nimbus manifest.
```

Comment:
Q: We say "Warning:" but should this be some fancy thing like above?

```md
:::warning
**This function will throw in Nightly and CI build** if you do not define `variableName` in the Nimbus manifest.
:::
```

Reply:
I find the warning UI is a little hard to read at the moment, I started a convo in #experimenter about it